atlas/operations/table.go

```
@@ -0,0 +1,303 @@
+
+	switch l {
+	case "GLOBAL": // global table; the default
+		c = c.ReplaceCommand("CREATE GLOBAL", "CREATE")
+	case "REGIONAL": // regional table
+		level = consensus.ReplicationLevel_regional
+		c = c.ReplaceCommand("CREATE REGIONAL", "CREATE")
+	case "LOCAL": // local table
+		level = consensus.ReplicationLevel_local
+		c = c.ReplaceCommand("CREATE LOCAL", "CREATE")
+	case "TABLE": // global table; the default
+		tableType = consensus.TableType_table
```

golangci-lint (1.62.2)

ineffectual assignment to tableType

(ineffassign)